frontend/components/Home/Dashboard/StudentCourses.tsx

```diff
@@ -44,11 +76,27 @@ const StudentCourses = (props: StudentCoursesProps) => {
             </Grid.Row>
 
             <Snackbar
-                open={success}
+                open={leaveToastOpen}
+                autoHideDuration={2000}
+                onClose={() => setLeaveToastOpen(false)}
+            >
+                <Alert
+                    severity={leaveToast.success ? "success" : "error"}
+                    onClose={() => setLeaveToastOpen(false)}
+                >
+                    <span>{leaveToast.message}</span>
+                </Alert>
+            </Snackbar>
+
+            <Snackbar
+                open={addToastOpen}
                 autoHideDuration={2000}
-                onClose={() => setSuccess(false)}
+                onClose={() => setAddToastOpen(false)}
             >
-                <Alert severity="success" onClose={() => setSuccess(false)}>
+                <Alert
```

Comment:
Instead of using 2 Alert components, you can share them using the pattern 
https://github.com/pennlabs/office-hours-queue/blob/c1e8facca8bff22bf37cdefe8c7d2481ad77fb85/frontend/components/Home/AccountSettings/AccountForm.tsx
here!

Reply:
oh wait my bad, I didn't see what you linked to here! Looks a lot cleaner than what I pushed just now... I'll do this instead (. ❛ ᴗ ❛.)